src/app/view/editor-main-view/data-views/editor.view.ts

```diff
@@ -32,6 +32,14 @@ import {UndoService} from "../../../services/data/undo.service";
 import {CopyService} from "../../../services/data/copy.service";
 import {StreckengrafikDrawingContext} from "../../../streckengrafik/model/util/streckengrafik.drawing.context";
 
+export enum ViewportOut {
+  ElmentIsInside,
```

Comment:
slight typo : `ElmentIsInside` -> `ElementIsInside`

do not forget to change here : `src/app/services/ui/ui.interaction.service.ts`

Reply:
Typo fixed